tests/test_requests.py

```diff
@@ -192,6 +192,23 @@ def test_token():
     assert request.token is None
 
 
+def test_content_type():
+    app = Sanic('test_content_type')
+
+    @app.route('/')
+    async def handler(request):
+        return text('OK')
```

Comment:
Could you also add an assert here as well? I think it'd be more useful to have the assert in the route handler than it would be outside of it.